alpha-blog/app/controllers/application_controller.rb

```diff
@@ -10,5 +10,12 @@ def logged_in?
     !!current_user
   end
 
+  def require_user
+    if !logged_in?
```

Comment:
```
unless logged_in?
```

Reply:
`unless`  is better?

Reply:
![image](https://user-images.githubusercontent.com/67599130/120484645-db5a7e00-c3e5-11eb-9e4d-965d238b8bd8.png)
How about this?
Better to do `unless current_user == @article.user` ?

Reply:
Personally, I prefer using `if`. But I bring this up here is to let you be aware of the existence of `unless` 
Your choice.